### DISCUSSION TITLE
lightdm-deepin-greeter 在编译出错

### DISCUSSION THREAD

**Discussion by Nethgul** [linuxdeepin/developer-center#2978] [Q&A | 问答板块]
![image](https://user-images.githubusercontent.com/42301063/122697732-8db09300-d278-11eb-927a-ab190e679d1a.png)
![image](https://user-images.githubusercontent.com/42301063/122697910-d9fbd300-d278-11eb-9bc7-3a9232ddd9d7.png)
这些头文件在那里获取,在UOS下并没有这些头文件,下载的工程中也没有包含相应的头文件

**Comment by Zeno-sole** [comment]
这些头文件在dde-qt-dbus-factory里面，你需要安装构建依赖才能正常编译
